html/about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/bee2670e70.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../css/about.css">
+    <title>About</title>
+  </head>
+  <header class="main-header">
+    <ul id="header-list">
+      <li class="header-item icon"><a href=""><i class="fa fa-facebook"></i></a></li>
+      <li class="header-item icon"><a href=""><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+      <li class="header-item"><a href="">English</a></li>
+      <li class="header-item"><a href="">My Page</a></li>
+      <li class="header-item"><a href="">Logout</a></li>
+    </ul>
+  </header>
+
+  <div class="mobile-nav">
+    <div class="hamburger">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+    </div>
+  </div>
+
+  <nav class="desktop-nav">
+    <img class="Logo" width="100" src="../images/page-logo-removebg-preview.jpg" alt="logo">
+    <ul class="nav-menu">
+      <li class="nav-items"> <a href="./index.html">Home</a></li>
+      <li class="nav-items"> <a href="./index.html">Categories</a></li>
+      <li class="nav-items"> <a href="#Events">Events</a></li>
+      <li class="nav-items"> <a href="#partner">Partner</a></li>
+    </ul>
+  </nav>
+
+
+
+  <section>
+    <div class="header">
+      <span class="subtitle">Welcome! To the Land of the Green</span>
+      <h1>Agro Food Summit 2022</h1>
```

Comment:
- [ ] The following image is bigger than the original please match the original design

![image](https://user-images.githubusercontent.com/75126481/177012979-91a0a5b0-b357-42d4-877e-e6fe7a028f13.png)